src/todd-coxeter-impl.cpp

```diff
@@ -119,8 +119,9 @@ The values in this enum can be used as the argument for
 should be performed.)pbdoc")
         .value(
             "hlt",
+            // JDM couldn't get the links to work in the following doc string.
```

Comment:
I think if the enums were written like tril, we could get this to work. I'll give it a go, and open a new PR if it works. Otherwise, I think this is fine.